### PR TITLE
Demo esp8266

### PR DESCRIPTION
**Pull request by mchindri**
Update old project to use noos wifi and adxl362 driver

### CODE REVIEW

File: projects/ADuCM3029_demo_esp8266/app_src/ADuCM3029_demo_esp8266.c [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+{
+	/**
+	 * Initialize managed drivers and/or services that have been added to
+	 * the project.
+	 * @return zero on success
+	 */
+	adi_initComponents();
+
+	if (ADI_PWR_SUCCESS != adi_pwr_Init())
+		return 0;
```
Comment (adrimbarean) [inline projects/ADuCM3029_demo_esp8266/app_src/ADuCM3029_demo_esp8266.c:73]: Usually return non-zero on error and 0 on success. But might want to use the power module for this.

**Comment by mchindri** [comment]
v2:
 - Rebase onto master:
 - fix astyle
 - replace number with define.
 - Fix return values
 - Add readme file

File: projects/ADuCM3029_demo_esp8266/app_src/ADuCM3029_demo_esp8266.c (hunk 1 above)
Comment (mchindri) [reply projects/ADuCM3029_demo_esp8266/app_src/ADuCM3029_demo_esp8266.c]: done

**Comment by adrimbarean** [comment]
Codacy has some issues

**Comment by mchindri** [comment]
v3:
 - Update for new no-OS makefile

**Comment by mchindri** [comment]
v4:
 - Rebase onto master
 - Add to .astyleignore some files.
 - Small updates

**Comment by mchindri** [comment]
Should old project: https://github.com/analogdevicesinc/EVAL-ADICUP3029/pull/53/commits/7ad19d3e709904e10738756d26a542aae521cf87
Be moved to a deprecated folder or just deleted?
There are no similarities with the current approach in order to do just an update.

**Comment by mchindri** [comment]
v5:
 - Remove old project

**Comment by mirceacaprioru** [comment]
Just remember to also update https://wiki.analog.com/resources/eval/user-guides/eval-adicup3029/reference_designs/demo_adxl362?s[]=esp8266

**Comment by mchindri** [comment]
> Just remember to also update [https://wiki.analog.com/resources/eval/user-guides/eval-adicup3029/reference_designs/demo_adxl362?s[]=esp8266](https://wiki.analog.com/resources/eval/user-guides/eval-adicup3029/reference_designs/demo_adxl362?s%5B%5D=esp8266)

It is already updated :)

**Review by dbogdan — APPROVED** [review]

**Pull request merged**